modules/Sources/Features/RecoveryPhraseDisplay/SecurityWarningStore.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  File.swift
+//  
+//
+//  Created by Praveen kumar Vattipalli on 09/09/23.
+//
+
+import Foundation
+import ComposableArchitecture
+import Models
+import Pasteboard
+
+public typealias SecurityWarningStore = Store<SecurityWarningReducer.State, SecurityWarningReducer.Action>
+
+public struct SecurityWarningReducer: ReducerProtocol {
+    public struct State: Equatable {}
+    
+    public enum Action: Equatable {
+        case debugMenuStartup
```

Comment:
Why is this action here, not used anywhere in the code except a view but never catched. Also confirm button doesn't sound like a button that should open a debug menu.